src/sonic-config-engine/sonic-cfggen

```diff
@@ -49,7 +49,8 @@ def main():
     parser=argparse.ArgumentParser(description="Render configuration file from minigraph data and jinja2 template.")
     parser.add_argument("-m", "--minigraph", help="minigraph xml file")
     parser.add_argument("-p", "--port-config", help="port config file, used with -m")
-    parser.add_argument("-y", "--yaml", help="yaml file that contains addtional variables")
+    parser.add_argument("-y", "--yaml", help="yaml file that contains addtional variables", action='append', default=[])
+    parser.add_argument("-Y", "--optional-yaml", help="optional yaml file that contains addtional variables", action='append', default=[])
```

Comment:
i feel this is error prone. can we do this check this outside sonic-cfggen?

like [ -f $file ] && sonic-cfggen -y $file.